exportoptionsgenerator/exportoptionsgenerator.go

```diff
@@ -136,12 +142,11 @@ func dependentApplicationBundleTargetsOf(exportMethod exportoptions.Method, appl
 		// 	<string>Development App Clip Profile</string>
 		// </dict>
 		// ..,
-		if exportMethod != exportoptions.MethodAppStore &&
-			target.ProductType == AppClipProductType {
+		if exportMethod != exportoptions.MethodAppStore && target.ProductType == AppClipProductType {
```

Comment:
note: Could use target.IsAppClipProduct(), once made public.

Reply:
Fixed, thank you!